tests/cases/fourslash/refactorConvertToGetAccessAndSetAccess2.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference path='fourslash.ts' />
+
+//// class A {
+////     /*a*/protected a: string;/*b*/
+//// }
+
+goTo.select("a", "b");
+edit.applyRefactor({
+    refactorName: "Generate 'get' and 'set' accessors",
+    actionName: "Generate 'get' and 'set' accessors",
+    actionDescription: "Generate 'get' and 'set' accessors",
+    newContent: `class A {
+    protected _a: string;
+    public get a(): string {
```

Comment:
If the initial property access is protected then this should be protected. The inner variable should be private